netbox/dcim/models/racks.py

```diff
@@ -173,8 +179,10 @@ def clean(self):
         super().clean()
 
         # Validate outer dimensions and unit
-        if (self.outer_width is not None or self.outer_depth is not None) and not self.outer_unit:
-            raise ValidationError(_("Must specify a unit when setting an outer width/depth"))
+        if (
+            self.outer_width is not None or self.outer_depth is not None or self.outer_height is not None
+        ) and not self.outer_unit:
+            raise ValidationError(_("Must specify a unit when setting an outer width/depth/height"))
```

Comment:
Trying to make it easier on translators
```suggestion
            raise ValidationError(_("Must specify a unit when setting an outer dimension"))
```